language/move-binary-format/src/layout.rs

```diff
@@ -132,7 +132,7 @@ impl TypeLayoutBuilder {
             Struct(shi) => MoveTypeLayout::Struct(StructLayoutBuilder::build_from_handle_idx(
                 m,
                 *shi,
-                type_arguments.to_vec(),
+                vec![],
```

Comment:
Here the type arguments readlly should be empty cuz it is a struct instantiation with no type arguements.